var/spack/repos/builtin/packages/stc/package.py

```diff
@@ -30,14 +30,14 @@ class Stc(AutotoolsPackage):
     """STC: The Swift-Turbine Compiler"""
 
     homepage = 'http://swift-lang.org/Swift-T'
-    url      = 'http://swift-lang.github.io/swift-t-downloads/stc-0.7.3.tar.gz'
+    url      = 'http://swift-lang.github.io/swift-t-downloads/spack/stc-0.0.0.tar.gz'
 
-    version('0.7.3', '6bf769f406f6c33d1c134521373718d3')
+    version('0.8.2', '883b0657f1aac9b81158ef0a8989be4c')
 
     depends_on('java')
     depends_on('ant')
     depends_on('turbine')
-    depends_on('zsh', type='run')
+    depends_on('zsh')
```

Comment:
I'm guessing that many of these are either `'build'` or `('build', 'run')` dependencies and not `('build', 'link')` as omitting the `type` implies.  Is that correct?